pickle2graphite.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, metricsDict, base):
                 (year, month, day) = devAttrs["Date"].split("-")
                 (hour, minute, second) = devAttrs["Time"].split(":")
             except KeyError:
-                raise ValueError("Date or Time is missing or incorrrectly formatted for this set of metrics")
-                raise
```

Comment:
I think it is worth letting this one stay as a log warning : 

*  The other changes which replace `ValueError`s with parsable (but unlikely) `"Date"` and `"Time"` values mean it should never happen unless something like say a solaredge message being garbled by a transmission error has resulted in a truncated json string (unlikely but not impossible), and
* It probably relates to only one instance of a device message (or at worst only the messages from one device type) in a stream of deciphered messages.  I use pickle2graphite.py (and prior to that a modified version of se2graphite.py) to follow the output of a `tcpdump | semonitor | pickle2graphite` pipe which I keep running continuously in (pseudo) daemon mode on my RPi, so it's better if pickle2graphite complains but carries on, rather than falls over when it stumbles.